src/current/v23.1/connection-pooling.md

```diff
@@ -25,37 +25,151 @@ Each time an application reads or writes data, it will request one of the connec
+
 {{site.data.alerts.callout_success}}
 To read more about connection pooling, see our [What is Connection Pooling, and Why Should You Care](https://www.cockroachlabs.com/blog/what-is-connection-pooling/) blog post.
 {{site.data.alerts.end}}
 
-## Sizing connection pools
+## Recommended settings for connection pools
+
+These are Cockroach Labs recommendations for settings common to most connection pooling software. If your application has multiple workers, each worker should be configured based on the kinds of queries required for its workload.
+
+### Set the maximum lifetime of connections
+
+The maximum lifetime of a connection should be set to no less than 5 minutes and no more than 30 minutes. Applications that connect to {{ site.data.products.serverless }} clusters should set this to 30 minutes, while applications that connect to {{ site.data.products.dedicated }} and {{ site.data.products.core }} clusters should set this to 5 minutes.
+
+### Set the maximum number of open connections
+
+The maximum number of open connections should be set to a value that is greater than the maximum number of concurrent queries that your application worker expects to make. Doing so minimizes the need to establish new connections.
+
+### Set the maximum number of idle connections
+
+Set the maximum number of idle connections to the same value as the maximum number of open connections.
+
+### Set the maximum lifetime of idle connections
+
+Set the maximum lifetime of idle connections to the same value as the maximum lifetime of connections.
+
+## Optimize your connection pool
+
+Follow these additional recommendations optimize the performance of your connection pool software.
+
+### Avoid spikes in new connections
+
+If possible configure your connection pool to avoid periodic spikes in new connections, also known as "thundering herds" or "connection storms." Most connection pools offer the ability to stagger their connection age through the use of a connection lifetime "jitter," where the wait time between new connections is randomized. Using a connection jitter of 10% of the maximum connection lifetime smooths out the rate of new connections to a cluster. For example, for a maximum connection lifetime of 300 seconds, set the connection jitter to 30 seconds.
+
+### Validate connections in a pool
+
+After a connection pool initializes connections to CockroachDB clusters, those connections can occasionally break. This could be due to changes in the cluster topography, or rolling upgrades and restarts, or network disruptions.
+
+Validating connections is typically handled automatically by the connection pool. For example, in HikariCP the connection is validated whenever you request a connection from the pool, and the `keepaliveTime` property allows you to configure an interval to periodically check if the connections in the pool are valid. Whatever connection pool you use, make sure connection validation is enabled when running your application.
```

Comment:
Since we're mentioning HikariCP and one of its config properties, suggest linking to that info.  IDK if this is the best place, but I found it mentioned in the project README: https://github.com/brettwooldridge/HikariCP#frequently-used